packages/icoutils/build.sh

```diff
@@ -0,0 +1,12 @@
+TERMUX_PKG_HOMEPAGE=https://www.nongnu.org/icoutils/
+TERMUX_PKG_DESCRIPTION="Extracts and converts images in MS Windows(R) icon and cursor files."
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=0.32.3
+TERMUX_PKG_REVISION=3
```

Comment:
Please remove this line, there's no need to initialize TERMUX_PKG_REVISION to `3`

Reply:
Done.

Reply:
Great, thanks!